CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
Your commit title must be at the imperative, present tense. Can be `implement front tests for ashley's editor`.
The gitmoji used is not relevant IMO, you are deploying nothing. I think ✨ is a more appropriated.

Reply:
ok noted, thanks for the reminder